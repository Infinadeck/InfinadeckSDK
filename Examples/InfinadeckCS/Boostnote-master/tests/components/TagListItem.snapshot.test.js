import React from 'react'
import renderer from 'react-test-renderer'
import TagListItem from 'browser/components/TagListItem'

it('TagListItem renders correctly', () => {
  const tagListItem = renderer.create(<TagListItem name='Test' handleClickTagListItem={jest.fn()} />)

  expect(tagListItem.toJSON()).toMatchSnapshot()
})
