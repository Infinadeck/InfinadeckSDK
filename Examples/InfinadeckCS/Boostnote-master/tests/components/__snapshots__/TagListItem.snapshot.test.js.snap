// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagListItem renders correctly 1`] = `
<div
  className="tagList-itemContainer"
>
  <div
    className="tagList-itemNarrow"
  />
  <button
    className="tagList-item"
    onClick={[Function]}
  >
    <span
      className="tagList-item-name"
    >
      # Test
      <span
        className="tagList-item-count"
      />
    </span>
  </button>
</div>
`;
